### PR TITLE
Build deb package on github CI

### PR DESCRIPTION
**Pull request by luqasz**

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -2,7 +2,7 @@ Source: py-libzfs
 Section: contrib/python
 Priority: optional
 Maintainer: William Grzybowski <william@grzy.org>
-Build-Depends: debhelper-compat (= 12),
+Build-Depends: debhelper,
```
Comment (william-gr) [inline debian/control]: Is this change really needed? I working as a debian maintainer and all developers tell me using debhelper-compat is the correct way to do this

File: sources.list [hunk 2]
```diff
@@ -0,0 +1,4 @@
+deb http://deb.debian.org/debian buster main contrib non-free
```
Comment (william-gr) [inline sources.list]: Should we use "stable" instead of "buster" so hopefully we dont have to worry about it on the next release?

**Review by william-gr — COMMENTED** [review]
Thanks, that is much appreciated.

I have only a couple questions/concerns.

File: sources.list (hunk 2 above)
Comment (luqasz) [reply sources.list]: Good point

File: debian/control (hunk 1 above)
Comment (luqasz) [reply debian/control]: I've changed it since apt says that package [does not exist](https://packages.debian.org/search?searchon=names&keywords=debhelper-compat). Only [debhelper](https://packages.debian.org/buster/debhelper).

Comment (william-gr) [reply debian/control]: I think debhelper-compat is handled especially by dpkg-buildpackage, its not an actual package

**Comment by luqasz** [comment]
I'd like to point that I've setup docker image in my docker hub account. Please build and push image yourself since my account may vanish in some time.

File: debian/control (hunk 1 above)
Comment (luqasz) [reply debian/control]: https://github.com/luqasz/py-libzfs/commit/86acf15a924c5af9a206fbeb88cb063f05df45b6/checks?check_suite_id=404470318

`dpkg-buildpackage` complains about `debhelper-compat`

nvm. I had to remove `debian/compat` file

**Review by william-gr — APPROVED** [review]
Perfect, we will set up our docker image once we find some spare time.

Thank you for your contribution

**Comment by william-gr** [comment]
@luqasz the merged work seems to be failing and I cant figure out why. Do you have access to look at the action job?

**Comment by luqasz** [comment]
I have only a big X and `This check failed`

**Comment by william-gr** [comment]
It says:

The following team project collection is stopped: 0aKIY6An43v3zj4IrNsOIkd50Men0piXIIiriqy9y5vGl4ZMnU. Start the collection and then try again. Administrator Reason: Creating Collection Host

**Comment by luqasz** [comment]
WOOT. Never seen that before. Try to add some empty line just to trigger github actions
